app/views/candidate_interface/volunteering/review/show.html.erb

```diff
@@ -2,22 +2,24 @@
 <% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
+  <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
       <%= t('page_titles.volunteering.short') %>
     </h1>
 
-    <%= render(CandidateInterface::VolunteeringReviewComponent.new(application_form: @application_form)) %>
-
-    <%= govuk_button_link_to t('application_form.volunteering.another.button'), candidate_interface_new_volunteering_role_path, class: 'govuk-button--secondary' %>
-
-    <%= form_with model: @application_form, url: candidate_interface_complete_volunteering_path do |f| %>
-      <div class='govuk-form-group'>
-        <%= f.hidden_field :volunteering_completed, value: false %>
-        <%= f.govuk_check_box :volunteering_completed, true, multiple: false, label: { text: t('application_form.volunteering.review.completed_checkbox') } %>
-      </div>
-
-      <%= f.govuk_submit t('continue') %>
+    <%= govuk_button_link_to nil, candidate_interface_course_choices_choose_path, class: 'govuk-button--secondary' do %>
+      <%= @application_form.application_volunteering_experiences.any? ? t('application_form.volunteering.another.button') : t('application_form.volunteering.add.button') %>
```

Comment:
There's a neat trick my last team used to handle these kind of conditions within the locale file itself by doing:

```suggestion
      <%= t('application_form.volunteering.add_button', @application_form.application_volunteering_experiences.any? %>
```

And then in the locale file doing:

```yaml
add_button:
  false: Add a role
  true: Add another role
```

Worth exploring?

Reply:
It’s a nice idea… however tempted to leave this as really there shouldn’t ever be a ‘Add a job’ button on this page, (as @adamsilver noted, the playback of a page with no work history/no volunteering should show the answer to the question, which you can then change)

Reply:
I'm not 100% convinced by never needing an "Add a job" button, even though in the prototype I've now implemented this by redirecting back to the first branching question if you have no jobs left after deleting one. It feels a bit weird though, and I think the more expected thing would be to return to the "You've said you have some work history, but you've not yet added any jobs" state, with some other mechanism to change your answer to the branching question.

Equally perhaps you ought to be able to change your answer to the branching question without having to delete all the jobs you've added first? But that feels even less likely.

Gah, it's so tricky!